Sage/carbon
# fix(tabs): move onclick handler to button element

fix #4883

### Proposed behaviour


Moved onclick handler up to button element from tab title content

### Current behaviour


onclick handler is on span element rather than button

### Checklist




#### QA


### Additional context



### Testing instructions
Test tab stories to ensure functionality still works.


The following CodeSandbox is an example of the broken behaviour.
You can see the new behaviour by looking at the version in the comment by `codesandbox[bot]`.
